internal/service/sagemaker/sagemaker_test.go

```diff
@@ -46,31 +46,49 @@ func TestAccSageMaker_serial(t *testing.T) {
 			"kernelGatewayAppSettings":                 testAccDomain_kernelGatewayAppSettings,
 			"kernelGatewayAppSettings_customImage":     testAccDomain_kernelGatewayAppSettings_customImage,
 			"kernelGatewayAppSettings_lifecycleConfig": testAccDomain_kernelGatewayAppSettings_lifecycleConfig,
-			"kernelGatewayAppSettings_defaultResourceAndCustomImage":  testAccDomain_kernelGatewayAppSettings_defaultResourceSpecAndCustomImage,
```

Comment:
change in this file seems unrelated to this PR

Reply:
Fixed.